datamodel_code_generator/__main__.py

```diff
@@ -73,6 +77,23 @@ class Exit(IntEnum):
     KeyboardInterrupt = 2
 
 
+namespace = Namespace(no_color=False)
+
+
+class SortingHelpFormatter(HelpFormatter):
+    def _bold_cyan(self, text: str) -> str:
+        return f'\x1b[36;1m{text}\x1b[0m'
+
+    def add_arguments(self, actions: Iterable[Action]) -> None:
+        actions = sorted(actions, key=attrgetter('option_strings'))
+        super().add_arguments(actions)
+
+    def start_section(self, heading: Optional[str]) -> None:
+        return super().start_section(
+            heading if namespace.no_color or not heading else self._bold_cyan(heading)
+        )
```

Comment:
@howardj99 
Could you please add unittest for the sections?

Reply:
@koxudaxi added